5001-6000/LIT5673AkkonuColl.xml

```diff
@@ -209,10 +212,28 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 </div>
 
                 <div type="textpart" xml:id="AkkColl6">
-                    <label>Miracle of the monk Isaac</label>
-
-                    <div type="textpart" xml:id="AkkMir6" corresp="LIT3592Miracle">
-                        <label>Miracle</label>
+                    <label>Miracle of the monk Yǝsḥaq</label>
+                    <note> Two variant versions of this miracle, with the same recipient but
+                        significant thematic differences, exist. To avoid confusion and overlap, the
+                        miracle has been subdivided into two distinct variants. </note>
+
+                    <div type="textpart" xml:id="AkkMir6A" corresp="LIT3592MiracleA">
```

Comment:
note that if you change the xml:id here you have to make sure that in manuscripts or other records where you may have referred to `LIT5673AkkonuColl#AkkMir6` will have to be updated to point to `LIT5673AkkonuColl#AkkMir6A` or `LIT5673AkkonuColl#AkkMir6B`, respectively. To avoid possible errors it is best to keep `AkkMir6 `for the first version and only add `AkkMir6B` or even have AkkMir6 for a non further identifiable miracle of Yeshaq, when one is not sure which one is included, and add AkkMir6A and AkkMir6B for the specific stories.

Even more importantly - concerning  `LIT3592Miracle`, you cannot simply rename CAe IDs or have two works associated with the same number (3592 = one number = one ID).

So you either, as above, use 3592 for the first version and create a NEW CAe ID for the second version, or keep LIT3592Miracle for the generic AkkMir6 (miracle of Yeshaq, any of the two, when one does not know which) and create TWO NEW CAe IDs for the two specific miracles.